R/combineSNV_CNA.R

```diff
@@ -0,0 +1,68 @@
+# combineSNV_CNA.R
+
+#' Combine mutation data from CNA and SNV
+#'
+#' \code{combineSNV_CNA} loads the vector of preprossed dataset from filtered CNA or SNV files,
+#'  creates hash keys corresponding to the input file type, combine and save the hashed
+#'  keys to a gX format file as output file.
+#'
+#' Details.
+#' @section gX files:
+#'  gX files contain data of SNV and CNA combined for each gene as one object, sorted per
+#'  gene. Tab-delimited text file with header.
+#'
+#'
+#' @param fName A vector of path of fully qualified file names of SNV and/or CNA files.
+#' @param fgX The fully qualified filename/path of a gX output file.
+#' @param silent Boolean option for writing combiling process information to console.
+#' @param noLog Boolean option for log results.
+#' @return number of hashed keys stored to the gX output file.
```

Comment:
I would not return a value that is not requested by the downstream function. Save file and log output as side-effects should be sufficient.